frontend/app/templates/components/mapbox/current-mouse-position.hbs

```diff
@@ -1,4 +1,6 @@
 {{!-- binds a map event with an action using mapbox-gl's on method --}}
-{{mapbox-gl-on this.mapboxEventName (action 'handleEvent') eventSource=map.instance}}
+{{mapbox-gl-on 'mousemove' (action 'handleMouseMove') eventSource=map.instance}}
```

Comment:
heads up @julialucyhogan  and I also headed in the direction of generalizing current-mouse-position. We made a `current-mouse-event` component as a part of #279 . Later when we merge in #279 we may need to reconcile all the different patterns

Reply:
@godfreyyeung cool, sounds like we could use your more generic approach. let's reconcile later. thanks